src/main/java/edu/kis/powp/jobs2d/command/ComplexCommand.java

```diff
@@ -0,0 +1,22 @@
+package edu.kis.powp.jobs2d.command;
+
+import java.util.Vector;
+
+public class ComplexCommand implements DriverCommand {
+    private Vector<DriverCommand> commandSeq = new Vector<DriverCommand>();
```

Comment:
List type (attribute and in the constructor) would be sufficient and would not limit the implementation.